core/bench_test.go

```diff
@@ -238,7 +218,7 @@ func makeChainForBench(db ethdb.Database, full bool, count uint64) {
 	var hash common.Hash
 	for n := uint64(0); n < count; n++ {
 		header := &types.Header{
-			Coinbase:    constants.BlackholeAddr,
+			Coinbase:    common.Address{},
```

Comment:
hmm this might fail benchs as we do have a coinbase verification in subnet-evm.

Reply:
The benchmarks are currently failing due to another issue (passing genesis), which I suggest we patch along with the update from upstream.
Since we only call `chain.GetHeaderByNumber` it actually doesn't verify the coinbase